docs/user/filesystem/commandline.rst

```diff
@@ -0,0 +1,175 @@
+GeoMesa FileSystem CommandLine Tools
+====================================
+
+Building from Source
+--------------------
+
+To build from source make sure you have Maven, Git, and Java SDK 1.8 installed. To build just the GeoMesa FileSystem
+distribution do this:
+
+.. code-block:: bash
+
+    git clone git@github.com:locationtech/geomesa.git
+    cd geomesa
+    build/mvn clean install -pl geomesa-fs/geomesa-fs-dist -am -T4
+
+The tarball can also be downloaded from geomesa.org.
+
+Installation
+------------
+
+To install the command line tools simply untar the distribution tarball:
+
+.. code-block:: bash
+
+    # If building from source
+    tar xvf geomesa-fs/geomesa-fs-dist/target/geomesa-fs_2.11-$VERSION-dist.tar.gz
+    export GEOMESA_FS_HOME=/path/to/geomesa-fs_2.11-$VERSION
+
+After untaring the archive you'll need to either define the standard Hadoop environmental variables or install hadoop
+using the ``bin/install-hadoop.sh`` script provided in the tarball. Note that you will need the proper Yarn/Hadoop
+environment configured if you would like to run a distributed ingest job to create files.
+
+If you are using a service such as Amazon Elastic MapReduce (EMR) or have a distribution of Apache Hadoop, Cloudera, or
+Hortonworks installed you can likely run something like this to configure hadoop for the tools:
+
+.. code-block:: bash
+
+    # These will be specific to your Hadoop environment
+    . /etc/hadoop/conf/hadoop-env.sh
+    . /etc/hadoop/conf/yarn-env.sh
+    export HADOOP_CONF_DIR=/etc/hadoop/conf
+
+After installing the tarball you should be able to run the ``geomesa-fs`` command like this:
+
+.. code-block:: bash
+
+    $ cd $GEOMESA_FS_HOME
+    $ bin/geomesa-fs
+
+The output should look like this:
+
+.. code-block::
+
+    INFO  Usage: geomesa-fs [command] [command options]
+      Commands:
+        ...
+
+Command Overview
+----------------
+
+configure
+~~~~~~~~~
+
+Used to configure the current environment for using the commandline tools. This is frequently run after the tools are
+first installed to ensure the environment is configured correctly::
+
+    $ geomesa-fs configure
+
+classpath
+~~~~~~~~~
+
+Prints out the current classpath configuration::
+
+    $ geomesa-fs classpath
+
+.. _fsds_ingest_command:
+
+ingest
+~~~~~~
+
+Ingest files into a GeoMesa FS Datastore. Note that a "datastore" is simply a path in the filesystem. All data and
```

Comment:
i was thinking of it more as a ummm suggestion style. like "Look for fun things" but i'll change it.